Extending Alerters Docs (#690)
zenml-io/zenml: docs/book/advanced-guide/alerts.md

---
description: Send automated alerts to chat services.
---

# :speech_balloon: Send Automated Chat Alerts

Many developer teams use real-time chat services like Slack for their

> day-to-day communication. Sometimes it can also be desired to have automated
> notifications be sent to those chat services from within ML pipelines,

Reviewer: ```suggestion
day-to-day communication. Members of those teams might also want to have automated
notifications be sent to those chat services from within ML pipelines,
```
